Sessions/session.ts

```diff
@@ -0,0 +1,44 @@
+const express = require('express')
+const app = express()
+const port = 3000
+const axios = require('axios');
+const jwt = require("jsonwebtoken")
+const bodyParser = require('body-parser')
+
+const jsonParser = bodyParser.json()
+const urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+app.post('/token', urlencodedParser, async (req, res) => {
+
+    const userId = req.body.userId
+    const passId = req.body.passId
+
+    // let response = await axios.post('/login', {
+    //     'userId': userId,
+    //     'passId': passId
+    // })
+
```

Comment:
Once you've setup your docker stuff, you can just communicate to my service here with the following url: **http://accounts:3000/login** and attach the username & password to the authorization header of the http request. You can take a look at my README to see what the response is so you can handle them appropriately in your own service.